src/Stryker.Core/Stryker.Core/StrykerRunner.cs

```diff
@@ -156,7 +159,7 @@ private IProjectComponent AddRootFolderIfMultiProject(IEnumerable<IProjectCompon
         {
             if (projectComponents.Count() > 1)
             {
-                var rootComponent = new FolderComposite
+                var rootComponent = new CsharpFolderComposite
```

Comment:
The CsharpFolderComposite is a non-generic FolderComposite that is only valid for csharp projects. This change means that when someone wants to improve on the work that was started for fsharp they will have to reinvent a way to make this generic so the StrykerRunner can work with both csharp and fsharp projects in a generic way.